source/developer/core-developer-handbook.md

```diff
@@ -57,6 +57,44 @@ Core developers are expected to code review pull requests submitted to the Matte
 - Unstable repositories that are still in development may be merged with a single approved review
 - Trivial changes, such as PM-approved text changes or typo fixes, maybe merged with only a single approved review for both stable and unstable repositories
 
+### Release Cutting Process ###
+
+**Note: To cut a build you need access to matterbuild. Please ask Christopher for access if you don't have it.**
+
+On code complete day, work with the PM on rotation to get all the pull requests for the current release merged into `master`. Once that is done and you've confirmed with the PM, cut the first release candidate by following these steps:
+
+1. Submit a PR to uncomment the upgrade code for the release version and add it to the version array. Use these PRs as examples, [https://github.com/mattermost/platform/pull/6336/files](https://github.com/mattermost/platform/pull/6336/files) and [https://github.com/mattermost/platform/pull/6600/files](https://github.com/mattermost/platform/pull/6600/files).
+2. Once the above PR is merged, post `/mb cut X.X.X-rc1` into a private channel. Replace `X.X.X` with the release version, ex: `3.10.0`. This will begin cutting the build and make an automatic post to the Release Disucssion channel.
```

Comment:
Yep, I recommend running it in a private channel because it the command comes back with real posts instead of ephemeral posts